### PR TITLE
fixes benchmark CE

### PR DESCRIPTION
**Pull request by ruslansennov**
mvn clean test -Pbenchmark

```
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] /home/ruslan/proj/javaslang/javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkPerformanceReporter.java:[393,13] found raw type: org.openjdk.jmh.results.Result
  missing type arguments for generic class org.openjdk.jmh.results.Result<T>
[INFO] 1 warning
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/ruslan/proj/javaslang/javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkPerformanceReporter.java: warnings found and -Werror specified
[INFO] 1 error
```

My environment:

```
$ mvn -version
Apache Maven 3.3.9
Maven home: /usr/share/maven
Java version: 1.8.0_91, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.4.0-22-generic", arch: "amd64", family: "unix"
```

/cc @paplorinc @eduardmanas

### CODE REVIEW

**Comment by eduardmanas** [comment]
@ruslansennov Good catch!!!

I remember having fixed that issue, so I have spent a little while scratching my head how on earth is not in master!

As it turns out, I fixed it in the branch of the other pull request https://github.com/javaslang/javaslang/pull/1317

**Comment by ruslansennov** [comment]
It seems that Daniel's [phrase](https://github.com/javaslang/javaslang/issues/1347) about "the PRs need to be as small as possible" is **very** actual :smile:

**Comment by danieldietrich** [comment]
Hi all, let's leave the current PRs as-is. I will finish my review soon. After that a PR should be a small, focused unit of work.

File: javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkPerformanceReporter.java [hunk 1]
```diff
@@ -390,7 +390,7 @@ public static TestExecution of(BenchmarkResult benchmarkResult, double outlierLo
         }
 
         public TestExecution(BenchmarkResult benchmark) {
-            Result primaryResult = benchmark.getPrimaryResult();
+            Result<?> primaryResult = benchmark.getPrimaryResult();
```
Comment (paplorinc) [inline javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkPerformanceReporter.java]: :+1:

**Pull request closed without merging**